doc/dev_guide/writing_extensions.rst

```diff
@@ -421,3 +421,8 @@ at the time of writing.
 The pre-release packages are obtained from `pypi <https://pypi.org>`_ and these
 will be used for any dependency which provides a pre-release package on pypi.
 
+A similar `Integration test  <https://github.com/hyperspy/hyperspy/actions>`__
```

Comment:
Looks like there are two underscores here?

Reply:
Yes, this is rst syntax, I don't remember the details but this syntax avoid issue/warning where there is several time the same link. Haven't check if this was really necessarily here, but I copied and paste it from somewhere else and I thought it would safer.